README.md

```diff
@@ -468,10 +468,10 @@ error will be displayed below the field. Rails normally wraps the fields in a
 div (field_with_errors), but this behavior is suppressed. Here's an example:
 
 ```html
-<div class="form-group has-error">
-  <label class="control-label" for="user_email">Email</label>
+<div class="form-group has-danger">
```

Comment:
As @GBH pointed out, `has-danger` isn't in Bootstrap 4 anymore. We should either fix the PR, or merge and remember to fix. Whatever is easier.

Reply:
I vote for merge and fix later or this is going to be sitting here for weeks.